.github/workflows/release.yml

```diff
@@ -16,7 +16,8 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-latest, windows-latest, macos-latest]
-        python: ["8", "9", "10"] # 3.x
+        # corresponds to python 3.9, 3.10, 3.11
+        python: ["9", "10", "11"]
```

Comment:
In an ideal world, we would be using the full python versions here, but since manipulation of string variables in GHA is difficult I think adding this comment to make things more easily grep-able is a fair compromise